.env.example

```diff
@@ -59,3 +59,8 @@ EMAIL_HOST_USER= # sender's email-id
 # passwords. More on that here:
 # https://dev.to/abderrahmanemustapha/how-to-send-email-with-django-and-gmail-in-production-the-right-way-24ab
 EMAIL_HOST_PASSWORD=
+
+# The driver for sending SMSs. Possible values are `sns` or `log`.
+# Use `sns` to have AWS SNS support. The AWS credentials must be present for this.
+# Use `log` to log SMSs into a file instead. Recommended for development mode.
+SMS_DRIVER='sns'
```

Comment:
you'll need to update .ENV.md - that's where we have been keeping the main details for each env variable